openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -163,13 +175,32 @@ func (o *DiscoverEtcdInitialClusterOptions) Run() error {
 		etcdInitialClusterEntries = append(etcdInitialClusterEntries, fmt.Sprintf("%s=%s", member.Name, member.PeerURLs[0]))
 	}
 	if len(targetMember.Name) == 0 {
+		archiveDataDir(filepath.Clean(o.DataDir))
```

Comment:
this won't work.  You cannot move the mount itself.  You'll need to move the content of mount.